app/utils.py

```diff
@@ -1188,8 +1188,7 @@ def __init__(self):
             # config.get returns unicode, but hmac is going to want a str
             self._key = configured_key.encode('utf-8')
         else:
-            configured_key = ''.join([
-                XsrfTool.TOKEN_CHARACTER_SET for _ in range(20)])
```

Comment:
Remove definition of TOKEN_CHARACTER_SET which is no longer used.

Reply:
Done in PR #562.